pkg/buffer.go

```diff
@@ -198,18 +200,16 @@ func (b *Buffer) buildReceptionReport() rtcp.ReceptionReport {
 		TotalLost:          lost,
 		LastSequenceNumber: extMaxSeq,
 		Jitter:             uint32(b.jitter),
-		LastSenderReport:   uint32(b.lastSRNTPTime >> 16),
+		LastSenderReport:   uint32(lastSRNTPTime >> 16),
 		Delay:              dlsr,
 	}
 	return rr
 }
 
 func (b *Buffer) setSenderReportData(rtpTime uint32, ntpTime uint64) {
-	b.mu.Lock()
-	defer b.mu.Unlock()
-	b.lastSRRTPTime = rtpTime
-	b.lastSRNTPTime = ntpTime
-	b.lastSRRecv = time.Now().UnixNano()
+	atomic.StoreUint32(&b.lastSRRTPTime, rtpTime)
```

Comment:
should we store this in a struct and use `StoreValue`? so they are all atomically set together